src/main/java/application/rest/v1/StatusEndpoint.java

```diff
@@ -73,6 +73,13 @@ public Response computeStatus(@Pattern(regexp = NAME_PATTERN_ONE_OR_MORE) @PathP
             @PathParam("resource-kind") @Parameter(description = "The Kubernetes resource kind for the resource") String kind,
             @Pattern(regexp = API_VERSION_PATTERN_ZERO_OR_MORE) @DefaultValue("") @QueryParam("apiversion") @Parameter(description = "The apiVersion of the resource") String apiVersion,
             @Pattern(regexp = NAME_PATTERN_ZERO_OR_MORE) @DefaultValue("default") @QueryParam("namespace") @Parameter(description = "The namespace of the resource") String namespace) {
+        if (Logger.isEntryEnabled()) {
+            Logger.log(StatusEndpoint.class.getName(), "computeStatus", Logger.LogType.ENTRY, 
```

Comment:
General comment:  need test cases.  Especially to validate that section and detail maps from other namespaces work and that pre-installed status/detail maps still work.

Reply:
I do have some test cases which are needed be cleaned up and checked in.